charts/mcpo/templates/deployment.yaml

```diff
@@ -0,0 +1,120 @@
+apiVersion: apps/v1
+kind: Deployment
```

Comment:
If you want to default to having persistence, wouldn't it make more sense for this to be a stateful set? That way you can scale this easier without needing shared filesystems.